### PR TITLE
PHP maintainer team

### PR DESCRIPTION
**Pull request by etu**
###### Motivation for this change
Since we now have support for maintainer teams I thought that we should have one for PHP. It's a big ecosystem used by many after all.

So I took the liberty to add a bunch of people and I will request a review from these people, and I won't merge them as part of the maintainer team unless they approve this PR.

These people are added to the maintainer team: @aanderse @globin @Ma27 @talyz 

If you **don't** want to be on the maintainer team, please leave a note and I'll update the PR.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Ma27 — APPROVED** [review]

File: nixos/tests/php/pcre.nix [hunk 1]
```diff
@@ -1,7 +1,9 @@
 let
   testString = "can-use-subgroups";
-in import ../make-test-python.nix ({ ...}: {
+in import ../make-test-python.nix ({pkgs, ...}: {
```
Comment (jtojnar) [inline nixos/tests/php/pcre.nix:3]: Why not just use `lib`?

**Review by aanderse — APPROVED** [review]

**Comment by etu** [comment]
@jtojnar I requested a review of you because I know you're active in the ecosystem as well, but for some reason I didn't add you to the team. Do you want in on the maintainer team?

File: pkgs/top-level/php-packages.nix [hunk 2]
```diff
@@ -486,6 +501,8 @@ in
 
       nativeBuildInputs = [ pkgs.pkgconfig ];
       buildInputs = with pkgs; [ cyrus_sasl zlib ];
+
+      meta.maintainers = pkgs.lib.teams.php.members;
```
Comment (talyz) [inline pkgs/top-level/php-packages.nix:505]: It should be possible to remove `pkgs` from all of these, since `lib` is already in scope.

**Review by talyz — APPROVED** [review]
Great stuff! Just one thing, but nothing crucial :)

**Comment by jtojnar** [comment]
I do not see myself actively maintaining these parts so I think I would prefer remaining an outside observer.

**Comment by Ma27** [comment]
@etu is there any reason this isn't merged yet? :)

**Comment by etu** [comment]
@Ma27 Yes, two reasons.
1) I haven't pushed the button
2) I gave the timeframe 20:00 CEST which is in 5 minutes :D

**Comment by etu** [comment]
@Ma27 Yes, one reason.

It would create horrible conflicts with the PHP refactor by @talyz. So I would rather rebase this one than he rebasing his changes.

But now I merged this (because I thought I was on his PR for a minute there) and then reverted this one to be able to merge his one without conflicts and will now rebase this one and make a new PR of it.

**Pull request merged**